sn_networking/src/cmd.rs

```diff
@@ -355,19 +355,13 @@ impl SwarmDriver {
             }
             SwarmCmd::GetLocalStoreCost { key, sender } => {
                 cmd_string = "GetLocalStoreCost";
-                let record_exists = self
-                    .swarm
-                    .behaviour_mut()
-                    .kademlia
-                    .store_mut()
-                    .contains(&key);
-                let cost = if record_exists {
-                    NanoTokens::zero()
```

Comment:
Have we not lost the 0 cost if record exists here?

Reply:
no, will still return Zero cost within the inner calculation.